finetuning/evaluation/evaluate_instance_segmentation.py

```diff
@@ -35,9 +35,9 @@ def eval_instance_segmentation_with_decoder(dataset_name, prediction_folder, exp
 
 def main():
     args = get_default_arguments()
-
+    peft_kwargs = {"rank": args.peft_rank, "module": args.peft_module}
```

Comment:
@anwai98 I changed the way the way the peft arguments are passed in the evaluation because I thought this would make things more flexible for different methods. Not sure if this is the most elegant way to do it

Reply:
I agree. This would be the most elegant way to handle the current functionalities.